bigtop-packages/src/common/presto/presto-server.svc

```diff
@@ -0,0 +1,74 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TYPE="server"
+DAEMON="presto-${TYPE}"
+DESC="Presto ${TYPE}"
+EXEC_PATH="/usr/lib/presto/bin/launcher"
+SVC_USER="presto"
+WORKING_DIR="/usr/lib/presto"
+DAEMON_FLAGS="--launcher-config /usr/lib/presto/bin/launcher.properties --jvm-config /etc/presto/jvm.config --config /etc/presto/config.properties --data-dir /var/lib/presto --pid-file /var/run/presto/${DAEMON}.pid --launcher-log-file /var/log/presto/${DAEMON}.out --server-log-file /var/log/presto/${DAEMON}.log"
```

Comment:
Not sure if this is relevant for BigTop or not, but it's possible to set pretty much all of the log/data file locations in the Presto node.properties file (e.g. node.launcher-log-file), which wouldn't be picked up by this hard coding of locations.